CHANGELOG.md

```diff
@@ -5,9 +5,15 @@
 <!-- There is always Unreleased section on the top. Subsections (Added, Changed, Fixed, Removed) should be added as needed. -->
 
 ## Unreleased
+### Changed
+- **BC BREAK** | `UserRecommendation` now returns new format of response in `->getData()`, which is a list of `stdClass` instances.
+
 ### Fixed
 - Exceptions occurring during async request now generate rejected promise (as they should) and are no longer thrown directly.
 
+### Added
+- `UserRecommendation` now sends which item properties should be returned alongside with item_ids.
```

Comment:
Maybe something `like

     - Possibility to define in `UserRecommendation` command which item properties should be returned alongside with item_ids.